x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/DefaultEndPointsIT.java

```diff
@@ -194,7 +194,11 @@ private static void assertDefaultChunkingSettings(Map<String, Object> modelConfi
         );
     }
 
-    public void testMultipleInferencesTriggeringDownloadAndDeploy() throws InterruptedException {
+    public void testMultipleInferencesTriggeringDownloadAndDeploy() throws InterruptedException, IOException {
+        // Creating an inference endpoint to force the backing indices to be created to reduce the likelihood of the test failing
+        // because it's trying to interact with the indices while they're being created.
+        putModel("initial-model", mockCompletionServiceModelConfig(TaskType.SPARSE_EMBEDDING, "streaming_completion_test_service"));
```

Comment:
If you delete this endpoint right after creating it, will the backing indices still exist? If so, it would be cleaner to create then delete the endpoint so that you don't need to account for an extra endpoint hanging around in other tests.

Reply:
Yeah I'll give that a shot 👍